concepticondata/conceptlists/Gruehn-2008-200.tsv

```diff
@@ -0,0 +1,201 @@
+Gruehn-2008-200-152	152	schuldig	guilty	1682	GUILTY	21.776	2.167	2.167	2.917	2.667	2.167	2.792	2.479	6	6	6.083	5.75	6	5.917	5.958	2.667	2.417	3.917	3.083	2.542	3.5	3.021	4.083	3.917	3.333	3.583	4	3.458	3.729	1.833	1.833	1.417	1.833	1.833	1.625	1.729	2.917	2.917	3.083	3.333	2.917	3.208	3.063	1.667	2.2	1.75
+Gruehn-2008-200-153	153	schwach	weak	1601	WEAK	21.652	2.333	2.75	3.25	2.583	2.542	2.917	2.729	3.5	3.917	3	3.182	3.708	3.087	3.404	1.917	1.667	2.5	2.417	1.792	2.458	2.125	5.182	5.417	4.583	4.25	5.304	4.417	4.851	2.25	2.583	2.25	1.667	2.417	1.958	2.188	4.083	4.25	3.75	3.6	4.167	3.682	3.935	1.75	2.625	2
+Gruehn-2008-200-154	154	schweigsam	silent			1.221	3.083	3.083	4	4.083	3.083	4.042	3.563	3.833	3.5	3.083	2.909	3.667	3	3.34	3.917	4.25	4.333	4.25	4.083	4.292	4.188	4.667	4.083	4.75	4.75	4.375	4.75	4.563	3.083	3.583	3.455	3.833	3.333	3.652	3.489	3.917	3.417	3.667	4.083	3.667	3.875	3.771	3.267	2.938	1.333
+Gruehn-2008-200-155	155	sehnsuechtig	longing			2.079	4.583	5	4.417	5.083	4.792	4.75	4.771	3.75	4.083	4.833	4.083	3.917	4.458	4.188	2.167	2.333	3.167	3.083	2.25	3.125	2.688	4.417	5.333	3.667	3.583	4.875	3.625	4.25	4.917	5.167	3.25	4.167	5.042	3.708	4.375	2.083	2.25	2.167	2.583	2.167	2.375	2.271	4.563	4.5	1.917
+Gruehn-2008-200-156	156	sensibel	sensitive			4.887	6.25	5.167	4.917	4.5	5.708	4.708	5.208	3.083	3.167	3.333	3.636	3.125	3.478	3.298	4	3.167	3.333	3.583	3.583	3.458	3.521	3.25	3.417	4.75	3.167	3.333	3.958	3.646	6.333	5.667	5.364	4.583	6	4.957	5.489	3.083	2.917	3.083	2.636	3	2.87	2.936	5.188	5.25	1.625
+Gruehn-2008-200-157	157	sentimental	sentimental			1.715	4.667	3.917	4.083	3.417	4.292	3.75	4.021	3.25	4.083	3.5	4	3.667	3.739	3.702	2.667	2.333	2.917	3	2.5	2.958	2.729	3.917	3	4.75	2.333	3.458	3.542	3.5	5.25	4.583	4.583	3.583	4.917	4.083	4.5	3.417	3.583	3.5	3.583	3.5	3.542	3.521	4	4.063	2
+Gruehn-2008-200-158	158	sinnlich	sensual			2.453	6.75	6.333	5.667	5.75	6.542	5.708	6.125	2	2.25	3.583	4.583	2.125	4.083	3.104	3.917	3.75	3.167	3.917	3.833	3.542	3.688	5.5	5.833	5.333	5	5.667	5.167	5.417	5.917	5.5	4.667	4.75	5.708	4.708	5.208	2.583	2.167	2.083	2.5	2.375	2.292	2.333	5.938	6.125	1.583
+Gruehn-2008-200-159	159	skeptisch	sceptical			20.367	3.75	4	4.083	4.75	3.875	4.417	4.146	4.833	5.167	4.583	4.5	5	4.542	4.771	4.917	5.167	4.25	4.583	5.042	4.417	4.729	4.25	4.667	3.333	3.182	4.458	3.261	3.872	4.5	4.583	5.25	4.083	4.542	4.667	4.604	3.417	3.417	3.75	3.364	3.417	3.565	3.489	5	4.938	1.667
+Gruehn-2008-200-160	160	sorgfaeltig	careful			14.091	5.167	4.75	6.273	6	4.958	6.13	5.532	4.667	4.917	3.833	4.5	4.792	4.167	4.479	5.917	5.75	5.667	6.083	5.833	5.875	5.854	2.833	4.75	4	3.75	3.792	3.875	3.833	5.5	4.417	6.25	5.917	4.958	6.083	5.521	3.75	3.833	3.75	4	3.792	3.875	3.833	5.75	5.5	1.875
```

Comment:
`2758 CAREFUL`?

Reply:
Since the other German lists only map "vorsichtig" to CAREFUL, I'd leave it unmapped.